### ISSUE TITLE
voice interference

### ISSUE THREAD

**Issue by musikboden** [paulrosen/abcjs#621]
If 2 voices in the same system is close to each other they will overlap like this:

X:1
T:voice interference
L:1/4
%%score (1 2)
K:C
V:1 clef=treble stem=down
cdef
V:2 stem=up
g2f2

![Skärmavbild 2021-02-19 kl  08 24 36](https://user-images.githubusercontent.com/18282700/108471607-ea308100-728b-11eb-8783-efd747a4c6ac.png)

It is possible to fix with the "spacer" "y" like this:

X:1
T:voice interference
L:1/4
%%score (1 2)
K:C
V:1 clef=treble stem=down
cdef
V:2 stem=up
g2y/20f2

![Skärmavbild 2021-02-19 kl  08 31 52](https://user-images.githubusercontent.com/18282700/108472356-ee10d300-728c-11eb-9ce7-1c8ec1fe9089.png)

But it would be nice if it could behave like in chords

X:1
T:voice interference
L:1/4
%%score (1 2)
K:C
V:1
[cdfa]

![Skärmavbild 2021-02-19 kl  08 27 52](https://user-images.githubusercontent.com/18282700/108471920-5e6b2480-728c-11eb-9434-4453eaffd3ae.png)

**Comment by paulrosen** [comment]
Ok, this won't make the next release, but I'll try to get it in the one after.

**Comment by mercvre** [comment]
I am having the same problem, it happens also when two voices share the same note but different durations:

The following code:

```
%%score (1 2)
V:1 clef=treble
[A2d2f2a2]) [C2E2]  | (D  F2   F)  |
V:2 clef=treble
y4                  |  D4          | 
```

gives this:

<img width="260" alt="Screen Shot 2021-03-04 at 08 34 14" src="https://user-images.githubusercontent.com/44339274/109927973-3b496780-7cc5-11eb-8ee8-d2ac76497050.png">

but it should display as:

<img width="282" alt="Screen Shot 2021-03-04 at 08 41 32" src="https://user-images.githubusercontent.com/44339274/109928149-721f7d80-7cc5-11eb-8656-cfecf3c7f7d2.png">

There are actually some software that have a directive called "**%%combinevoices**" or "%%**voicecombine**" to choose how unisons behave with multiple voices, here is a link: http://moinejf.free.fr/abcm2ps-doc/index.html.

Also the same problem happen when different voices share the same note but with different accidentals:

```
%%score (1 2)
L:1/4
K:C
V:1 clef=treble
[=EGB]2 
V:2 clef=treble
_E _D
```

The previous code gives this:

<img width="175" alt="Screen Shot 2021-03-04 at 08 48 00" src="https://user-images.githubusercontent.com/44339274/109929026-62ecff80-7cc6-11eb-86cb-f919805ded31.png">

when it should actually display as:

<img width="190" alt="Screen Shot 2021-03-04 at 08 59 11" src="https://user-images.githubusercontent.com/44339274/109930474-fd9a0e00-7cc7-11eb-932c-ad6533479e09.png">

Hope this helps!

**Comment by paulrosen** [comment]
Fixed in version 6.0.0-beta.35

**Comment by musikboden** [comment]
> Fixed in version 6.0.0-beta.35

Great! :)
